src/printer.js

```diff
@@ -2106,11 +2122,29 @@ function printArgumentsList(path, options, print) {
     const shouldBreak = shouldGroupFirst
       ? printed.slice(1).some(willBreak)
       : printed.slice(0, -1).some(willBreak);
+
+    let printedLastArgExpanded;
+    if (shouldGroupFirst) {
+      printedLastArgExpanded = printed;
+    } else {
+      // We want to print the last argument with a special flag
+      let i = 0;
+      path.each(function(argPath) {
+        if (i++ === args.length - 1) {
+          printedLastArgExpanded = printed
+            .slice(0, -1)
+            .concat(path.call(
+              print.withArgs({expandLastArg: true})
```

Comment:
The second argument `options` are the printer options (printWidth, useTabs...) and not calling the function with specific arguments.

I thought about doing `{...options, expandLastArg: true}` but the issue is that this option is forwarded down recursively. So this would mean that I'd also have to make sure to unset the expandLastArg before forwarding down otherwise the entire sub-tree would have this flag set, instead of just the node.